caput/config.py

```diff
@@ -91,17 +91,16 @@ def __init__(self, default=None, proptype=None, key=None, deprecated=False):
         self.default = default
         self.key = key
         self.propname = None
-        # If this property is deprecated, we'll emit a warning
-        # when it's used
-        self._deprecated = deprecated
+        self.__deprecated = deprecated
```

Comment:
Why the double underscore?  Do you want it mangled?

Reply:
I was going back and forth on this. I had a weak argument that mangling could be useful since the user should never be trying to access the value of `deprecated` so why not, but it's also more pythonic to just make it private and I don't really think there's much of a case for extending this class in a way that could conflict with that attribute. So I can just revert that.